src/pages/tutorials/smart-rollups/optimize.md

```diff
@@ -0,0 +1,104 @@
+---
+id: optimize
+title: "Part 3: Optimizing the kernel"
+lastUpdated: 25th October 2023
+---
+
+To originate the kernel on Tezos, it must fit within the maximum size for a layer 1 operation.
+In these steps, you optimize the kernel:
```

Comment:
```suggestion
In these steps, you reduce the size of the kernel:
```

Optimize still feels like it would give the wrong impression to a developer - this section is all about `Part 3: Reducing the kernel size`, and includes an explanation of why.